src/api/api.js

```diff
@@ -156,7 +162,7 @@ function new_rpc_from_routing(routing_table) {
     }
 
     return new RPC({
-        APIClient,
+        new_client_factory,
```

Comment:
As I commented above, if possible I would try to pass to RPC a compiled new_client function that already includes the schema.

Reply:
Please see my comment above regarding the RPC reference.